### PR TITLE
Update use-aptos-cli.md

### PR DESCRIPTION
**Pull request by junkil-park**
Added the description of the `move prove` command of Aptos CLI.

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2117)

### CODE REVIEW

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md [hunk 1]
```diff
@@ -536,6 +536,20 @@ Test result: OK. Total tests: 2; passed: 2; failed: 0
 }
 ```
 
+### Proving Move
+
+The `aptos` CLI can be used to run [Move Prover](https://github.com/move-language/move/tree/main/language/move-prover) which is a formal verification tool for the Move language. The below example proves the `hello_prover` package in [move-examples](../../aptos-move/move-examples/).
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md:541]: Link broken for move-examples. Correct link is `[move-examples](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/hello_prover)`.

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md [hunk 2]
```diff
@@ -536,6 +536,20 @@ Test result: OK. Total tests: 2; passed: 2; failed: 0
...
+```bash
+SUCCESS proving 1 modules from package `hello_prover` in 1.649s
+{
+  "Result": "Success"
+}
+```
+
 ### Debug and Print Stacktrace
 
 In this example, we will use `DebugDemo` in [debug-move](./debug-move-example)
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md:555]: The `debug-move` link is broken. I don't know where this file is.

**Review by rajkaramchedu — APPROVED** [review]
Broken links. Approving so I am not blocking.

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md (hunk 1 above)
Comment (junkil-park) [reply developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md]: Got it. Fixed.

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md (hunk 2 above)
Comment (junkil-park) [reply developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md]: Fixed.

**Pull request merged**